app/upgrades/v1/mainnet/params_update.go

```diff
@@ -0,0 +1,123 @@
+package mainnet
+
+import (
+	"fmt"
+	"time"
+
+	"cosmossdk.io/math"
+	sdkmath "cosmossdk.io/math"
+	"github.com/babylonlabs-io/babylon/app/keepers"
+	appparams "github.com/babylonlabs-io/babylon/app/params"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var (
+	// Slashing params
+	MainnetMinSignedBlocks, _    = sdkmath.LegacyNewDecFromStr("0.6")
+	MainnetSlashFractionDowntime = sdkmath.LegacyZeroDec()
+	MainnetDowntimeJailDuration  = 300 * time.Second
+
+	// Governance params
+	MainnetVotingPeriod = 3 * 24 * time.Hour
+	// 50k BBN
+	MainnetMinDeposit = sdk.NewCoin(appparams.DefaultBondDenom, math.NewInt(50_000_000000))
+	// 200k BBN
+	MainnetMaxDepositPeriod      = 14 * 24 * time.Hour
+	MainnetExpeditedVotingPeriod = 24 * time.Hour
+	MainnetExpeditedMinDeposit   = sdk.NewCoin(appparams.DefaultBondDenom, math.NewInt(200_000_000000))
+
+	// Consensus params
+	MainnetBlockGasLimit = int64(300000000)
+	// Staking params
+	MainnetBabyMinCommissionRate, _ = sdkmath.LegacyNewDecFromStr("0.03")
+
+	// BTC checkpoint params
+	MainnetBTCConfirmationDepth = uint32(30)
+	MainnetBTCFinalizationDepth = uint32(300)
+
+	// Distribution params
+	MainnetCommunityTax          = sdkmath.LegacyZeroDec()
+	MainnetWithdrawalAddrEnabled = true
+)
+
+// MainnetParamUpgrade make updates to specific params of specific modules
+func MainnetParamUpgrade(ctx sdk.Context, k *keepers.AppKeepers) error {
+	// update slashing params
+	slashingParams, err := k.SlashingKeeper.GetParams(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to get slash params: %w", err)
+	}
+
+	slashingParams.MinSignedPerWindow = MainnetMinSignedBlocks
+	slashingParams.DowntimeJailDuration = MainnetDowntimeJailDuration
+	slashingParams.SlashFractionDowntime = MainnetSlashFractionDowntime
+
```

Comment:
store the updated params
```suggestion
if err := k.SlashingKeeper.Params.Set(ctx, slashingParams); err != nil {
		return fmt.Errorf("failed to set slashing params: %w", err)
	}
```